docs/src/index.md

```diff
@@ -8,7 +8,7 @@ end
 
 A multi-formalism simulator for noisy quantum communication and computation hardware with support for symbolic algebra, multiple simulation backends, a variety of noise models, discrete event simulation, optimization, and visualization.
 
-We are also preparing a [getting started manual](@ref manual).
+To install QuantumSavory, see: [getting started manual](@ref manual).
```

Comment:
manual.md is very short, to the point that it should probably just be embedded in this page itself, not a separate page.

It is difficult balance to strike, but I prefer to avoid the proliferation of many small pages -- that becomes much harder to navigate.